install_bun.sh

```
@@ -0,0 +1,320 @@
+    # Bold
+    Bold_Green='\033[1;32m' # Bold Green
+    Bold_White='\033[1m'    # Bold White
+fi
+
+error() {
+    echo -e "${Red}error${Color_Off}:" "$@" >&2
+    exit 1
+}
+
+info() {
+    echo -e "${Dim}$@ ${Color_Off}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)